topics:
    odm_sub: "/bebop/odom"
    cam_sub: "/bebop/image_raw"
    cam_compressed_sub: "/bebop/image_raw/compressed"
    hpe_zone_sub: "/stickman_cont_area"
    ui_pub: "/uav/gui"



